Libraries/LibC/netdb.cpp

```diff
@@ -459,11 +468,11 @@ struct protoent* getprotoent()
     __getproto_buffer.p_name = const_cast<char*>(__getproto_name_buffer.characters());
     __getproto_buffer.p_proto = __getproto_protocol_buffer;
 
-    __getproto_alias_list.clear();
-
-    for (auto& alias : __getproto_alias_list_buffer) {
-        __getproto_alias_list.append((char*)alias.data());
-    }
+    __getproto_alias_list.clear_with_capacity();
+    __getproto_alias_list.ensure_capacity(__getproto_alias_list_buffer.size() + 1);
+    for (auto& alias : __getproto_alias_list_buffer)
+        __getproto_alias_list.append(reinterpret_cast<char*>(alias.data()));
```

Comment:
Same `unchecked_append()` suggestion?

Reply:
Which other `unchecked_append()` suggestion you're referring to? :eyes:

But yeah, let me do that.

Reply:
huh. the other one got lost on the way.